### ISSUE TITLE
Documents created with Document Type Collection caplitalize the first letter of the alias

### ISSUE THREAD

**Issue by amirkhan81** [umbraco/Umbraco-CMS#3717]
### Bug summary
Clean install of Umbraco 7.12.0
When creating document types with the Document Type Collection method, the alias of the document types created will have a capitalized first letter in the alias where as creating a document type with the normal Document Type option creates an alias where the first letter is lowercase.

This might be confusing when trying to query document types by alias with Xpath and such.

**Comment by kjac** [comment]
@amirkhan81 how do you reproduce this?

I have a doc type collection on the latest codebase. The parent doctype is called "Collection Parent" and the item doctype is called "Collection Child". 

Given this view snippet:

```cshtml
@{
    var collectionParent = Umbraco.TypedContent(1123);
    <div>
    Collection parent alias: <strong>@collectionParent.DocumentTypeAlias</strong>
    <br/>
    Collection item alias: <strong>@collectionParent.Children().First().DocumentTypeAlias</strong>
    </div>
}
```
...where ID 1123 is a collection parent with one or more collection item children, this is the rendered result:

![image](https://user-images.githubusercontent.com/7405322/48736764-7afef900-ec4c-11e8-9889-9b8bc1e2bb87.png)

**Comment by amirkhan81** [comment]
When I create  new doctypes with the names Categories and Category on a fresh install of 7.12.4 (sorry had the wrong version before)

I get this output: 
Collection parent alias: Categories
Collection item alias: Category

![screen shot 2018-11-19 at 5 06 12 pm](https://user-images.githubusercontent.com/976383/48738175-25135c80-ec1e-11e8-90b8-cb85835c739b.png)
![screen shot 2018-11-19 at 5 06 16 pm](https://user-images.githubusercontent.com/976383/48738177-25135c80-ec1e-11e8-8371-aec9ba692f53.png)

**Comment by kjac** [comment]
@amirkhan81 I guess this must have been fixed (explicitly or implicitly) in the latest codebase. This is what I see:

![doctype-collection-alias-casing](https://user-images.githubusercontent.com/7405322/48756010-c68ac480-ec97-11e8-8968-4c31a7dffa97.gif)

**Comment by nul800sebastiaan** [comment]
Not yet, it will be released when 7.13.0 comes out soon, hopefully in the next few weeks. Thanks for the double-check @kjac !
